src/android/InAppBrowser.java

```diff
@@ -777,6 +809,11 @@ public String showWebPage(final String url, HashMap<String, String> features) {
             if (bannerMessageSet != null) {
                 bannerMessageText = bannerMessageSet;
             }
+
+            String showNavigationBarSet = features.get(SHOW_NAVIGATION_BAR);
+            if (showNavigationBarSet != null) {
+                showNavigationBar = showNavigationBarSet.equals("yes") ? true : false;
```

Comment:
My guess is that you don't need the ternary here and could just do:
```suggestion
                showNavigationBar = showNavigationBarSet.equals("yes");
```
but could be wrong.

Reply:
I copied the style of the above lines where we were setting other variables from the options, but yea it can definitely be simplified!